usecases/rule_snoozes.go

```diff
@@ -292,6 +294,8 @@ func (usecase RuleSnoozeUsecase) SnoozeDecision(
 		},
 	)
 
+	// TODO: do we really ignore errors here?
```

Comment:
@Pascal-Delange I am under the impression that we ignore any error coming out of the transaction. Is this on purpose?